gammapy/maps/region.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, region, axes=None, wcs=None, binsz_wcs="0.1 deg"):
         if wcs is None and region is not None:
             if isinstance(region, CompoundSkyRegion):
                 center = compound_region_center(region)
+                print(f"Center: {center}")
```

Comment:
We do not use log.info here?

Reply:
Oops, this `print` statement was not meant to stay....I'll remove it, thanks!